nixos/modules/system/boot/systemd.nix

```diff
@@ -604,6 +607,19 @@ in
         interval) that is applied to all messages generated on the system.
         This rate limiting is applied per-service, so that two services
         which log do not interfere with each other's limit.
+
+        Note that the effective rate limit is multiplied with a factor derived
```

Comment:
it tells me "by a factor"

Reply:
Ah, I hadn't grammar-checked that because I copy-pased that entire sentence straight from the `journald.conf` man page, which also has that incorrect grammar:

https://www.freedesktop.org/software/systemd/man/journald.conf.html

I agree that `multiplied by` is correct here, and will update it.

Reply:
Done.

Reply:
Upstreaming at https://github.com/systemd/systemd/pull/15552.